### ISSUE TITLE
Money symbol always displayed at end

### ISSUE THREAD

**Issue by ConnorJC3** [Arckenver/Nations#3]
```
> sponge plugins
[Server thread/INFO]: Plugins (13): Minecraft, Minecraft Coder Pack, Forge Mod Loader, Minecraft Forge, SpongeForge, DjxyCore, Nations, Nucleus, ServerListPlus, Total Economy, VoxelSniper-Forge, VoxelSniper-Sponge, WorldEdit

minecraft@sponge:~/sponge/mods$ ls
DjxyCore-1.0.2.jar     PermissionManager-1.2.jar.disabled          TotalEconomy-v1.4.1.jar
Nations-0.7.jar        ServerListPlus-3.4.5-Universal.jar          VoxelSniper-7.1.0-SNAPSHOT-all.jar
Nucleus-0.4.2-all.jar  spongeforge-1.8.9-1890-4.1.0-BETA-1320.jar  worldedit-forge-mc1.8.9-6.1.1-dist.jar
```

Even though I have TotalEconomy setup to display the symbol at the beginning, Nations always displays it at the end of the amount of currency. This is annoying as I'm trying to use US dollars as my currency, and the '$' is usually displayed at the beginning of the amount, not the end.

**Comment by Arckenver** [comment]
You're right, I'll be working on that.

Sorry for being late, and thanks for reporting that issue.

**Comment by Arckenver** [comment]
Fixed with commit 100dbebdab04cf12c542ad9db97481aaad5b9f83 and release of [v0.9](https://github.com/Arckenver/Nations/releases/tag/v0.9). Now money amount display is entirely up to the economy plugin.

_N.B. If an economy plugin doesn't support symbol order, Nations can't do anything about that._
